code/datums/achievements/antagmastery.dm

```diff
@@ -0,0 +1,112 @@
+/datum/award/achievement/antagmastery
+	category = "Antag Mastery"
+	reward = 400
+
+/datum/award/achievement/antagmastery/on_unlock(mob/user)
+	. = ..()
+	for(var/datum/award/achievement/antagmastery/A in subtypesof(/datum/award/achievement/antagmastery))
```

Comment:
I feel like this is going to slow the server very much when the cult consisting of every player on the server wins.
Can you implement cache on this?